ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -114,6 +114,8 @@ spec:
       params:
         - name: cert_project_id
           value: "$(tasks.certification-project-check.results.certification_project_id)"
+        - name: pyxis_url
```

Comment:
Copying the Alex comment:
```
Rather than hard coding the default workspace path, which could change in a future Tekton release, it'd be safer to pass the relative path within the workspace. From the task you can access the workspace path with some variable substitutions. E.g.:

$(workspaces.<workspaceName>.path)
$(workspaces.<workspaceName>.bound) (for optional workspaces)
@Allda may have already created a tracker for changing this in existing tasks. It'd be good to do for the new tasks in this PR though.
```